### DISCUSSION TITLE
Dialog Root is not respecting conditional size

### DISCUSSION THREAD

**Discussion by techwithtwin** [chakra-ui/chakra-ui#9348] [General]
![image](https://github.com/user-attachments/assets/0937fd7a-3a7b-4295-aa15-34cdb6db75ef)

The dialog root is not respecting the dialog root

![image](https://github.com/user-attachments/assets/faf58a14-d11f-474a-a72c-8f3dbe520318)

**Comment by segunadebayo** [comment]
Put `size` on `Dialog.Root`. Responsive values like `size={{ base: "sm", md: "xl" }}` work there.
